src/main/java/com/team4/testingsystem/exceptions/ControllerAdvice.java

```diff
@@ -0,0 +1,37 @@
+package com.team4.testingsystem.exceptions;
+
+import com.team4.testingsystem.dto.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@RestControllerAdvice
+public class ControllerAdvice extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler(value = {ConflictException.class})
+    @ResponseStatus(value = HttpStatus.CONFLICT)
+    public ErrorResponse handleConflictException(ConflictException e) {
+        return new ErrorResponse(e.getMessage());
+    }
+
+    @ExceptionHandler(value = {FileOperationException.class})
+    @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
```

Comment:
Maybe we should send 4xx responses only?

Reply:
What code should I use then?

Reply:
I think problems with file saving/deleting are server errors but not client

Reply:
`409 Conflict`? `417 Expectation Failed`?
`418 I'm a teapot` :)